app/policies/trainee_policy.rb

```diff
@@ -34,7 +34,7 @@ def lead_school_scope
     end
 
     def lead_partner_scope
```

Comment:
This looks good, however the `if` statement at [`#user_scope`](https://github.com/DFE-Digital/register-trainee-teachers/pull/4505/files#diff-528bccf8cbb54992c282220801aa53bf195c9520cb5ceb641aa5fece9aaf0fbfR19) would have to be refactored.